addons/mail/migrations/12.0.1.0/end-migration.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2018 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def fill_mail_tracking_value_track_sequence(env):
+    env.cr.execute(
+        """
+        SELECT DISTINCT mtv.field, mm.model
+        FROM mail_tracking_value mtv
+        INNER JOIN mail_message mm ON mtv.mail_message_id = mm.id
+        INNER JOIN ir_model_fields imf ON (
+            imf.name = mtv.field AND mm.model = imf.model) 
+        WHERE mm.model IS NOT NULL AND imf.track_visibility IS NOT NULL AND
+            imf.track_visibility != 'false'
+        """
+    )
+    for field, model in env.cr.fetchall():
+        if field in list(env[model]._fields.keys()):
```

Comment:
For avoiding errors do:
```suggestion
        if env.get(model) and field in list(env[model]._fields.keys()):
```